aexpect/remote.py

```diff
@@ -17,6 +17,12 @@
 from aexpect import rss_client
 
 
+#: prompt to be used for shell sessions on linux machines (default)
+PROMPT_LINUX = r"^\\[.*\\][\\#\\$]\\s*$"
+#: prompt to be used for shell sessions on windows machines
+PROMPT_WINDOWS = r"^\w:\\.*>\s*$"
```

Comment:
Useful

Reply:
Yes, it originates from earlier discussions one year ago and I think it was originally your suggestion (cannot recall any longer).